cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -386,33 +388,54 @@ ignoringSeverity :: Tracer IO a -> Tracer IO (WithSeverity a)
 ignoringSeverity tr = Tracer $ \(WithSeverity _ ev) -> traceWith tr ev
 {-# INLINE ignoringSeverity #-}
 
+countMyBlocks
+  :: forall blk. HasHeader (Header blk)
+  => AF.AnchoredFragment (Header blk)
+  -> Word64
+countMyBlocks = fromIntegral . length . AS.filter isMine
+  where isMine :: Header blk -> Bool
+        isMine _ = True
```

Comment:
Added a new `countMyBlocks` function.  All that's left is to implement `isMine`.